src/main/resources/templates/fragments/docUploadSeeWhatToAddReveal.html

```diff
@@ -1,15 +1,14 @@
 <th:block th:fragment="docUploadSeeWhatToAddReveal">
-  <th:block th:replace="~{'fragments/honeycrisp/reveal' :: reveal(
-                  controlId='r1',
-                  linkLabel=~{::revealLabel1},
-                  content=~{::revealContent1})}">
-    <th:block th:ref="revealLabel1">
+  <th:block th:replace="~{fragments/accordion :: accordion(
+    buttonLabel=~{::contentLabel},
+    content=~{::contentContent})}">
```

Comment:
The name :-(

Reply:
@vrajmohan  just updated this to just be "content".  I had been wondering about naming :)